CHANGELOG.md

```diff
@@ -2,6 +2,28 @@
 
 All notable changes to the "azure-cosmosdb" extension will be documented in this file.
 
+## 0.6.0 - 2018-04-05
+
+### Added
+
+- Mongo
+  - mongo.shell.path setting no longer required if shell is in system path
+  - Allow mongo.shell.path to be specified in workspace settings as well as user settings
+  - No longer asking for an id for new documents
+  - Multiple arguments (e.g. projections) now work for common commands in the scrapbook
```

Comment:
the multiple arguments one feels like a bug, not a feature add, but I don't have a strong preference.

For some of the more major bug fixes, it's also fine to list them out in the "Fixed" section instead of just linking to github